axon-framework/tuning/event-processing.md

```diff
@@ -0,0 +1,82 @@
+
+First, the instance of the Tracking Event Processor must be obtained. This can be done using Axon's Configuration API like so:
+
+```java
+// The `Configuration` was returned through the `Configurer` or is available as a bean in the Spring Application Context
+public TrackingEventProcessor retrieveTrackingProcessor(org.axonframework.config.Configuration axonConfig,
+                                                        String processorName) {
+    return axonConfig.eventProcessingConfiguration()
+                     .eventProcessor(processorName) // This call returns an Optional
+                     .filter(eventProcessor -> eventProcessor instanceof TrackingEventProcessor)
+                     .map(eventProcessor -> (TrackingEventProcessor) eventProcessor)
+                     .orElseThrow(() -> new IllegalStateException(
+                             "No Tracking Event Processor found with name " + processorName
+                     ));
+}
+```
+
+Using the above snippet, a split or merge can be called as follows:
+
+```java
+int segmentId;
+TrackingEventProcessor trackingProcessor = retrieveTrackingProcessor(axonConfig, processorName);
+
+// Split...
+CompletableFuture<Boolean> futureResult = trackingProcessor.splitSegment(segmentId);
+
+// Merge...
+CompletableFuture<Boolean> futureResult = trackingProcessor.mergeSegment(segmentId);
+```
+
+> **Multi instance set up**
+>
+> If you have several instances of a given Axon application, that regularly means you have duplicated your Tracking Event Processors. Such a set up is a regular scenario to require segment tuning.
+>
+> Note that, especially, in such a setup you would need to delegate said split or merge to the correct instance. The "correct instance", in this case, is the instance owning the segment you want to split and merge.
+
+## Blacklisting Events <a id="blacklisting-events"></a>
+
+In a heterogeneously distributed application landscape your event handling components might receive events they do not have actual event handling members for. That this occurs is completely fine; the chances of a single application handling the entirety of all existing events is very small. This fact however does open up the possibility for optimization by _blacklisting_ events.‌
+
+To this end Axon has to option to automatically blacklist events it cannot handle. The Tracking Event Processor takes the lead in actual blacklisting, which it does by signaling the utilized event stream when none of its handlers can handle the event in question. The event stream provided by the Axon Server connection in turn implements the functionality to notify an Axon Server node that certain events cannot be handled by it.‌
+
+By default, blacklisting is turned for an Axon client connected to Axon Server. To disable blacklisting, the `disableEventBlacklisting` property can be adjusted as follows:
```

Comment:
`By default, blacklisting is turned on for an Axon client connected to Axon Server` instead of `By default, blacklisting is turned for an Axon client connected to Axon Server`